core/src/main/java/dev/hypera/scaffolding/schematic/readers/SpongeSchematicReader.java

```diff
@@ -103,7 +100,7 @@ private void readBlockPalette(@NotNull Schematic schematic, @NotNull NBTCompound
         }
 
         Map<String, Integer> palette = unsortedPalette.entrySet().stream().sorted(Map.Entry.comparingByValue())
-            .collect(LinkedHashMap::new, (map, entry) -> map.put(entry.getKey(), entry.getValue()), LinkedHashMap::putAll);
+                .collect(LinkedHashMap::new, (map, entry) -> map.put(entry.getKey(), entry.getValue()), LinkedHashMap::putAll);
```

Comment:
This should only be indented with 4 spaces.

Reply:
I sepperated those lines more to be more readable & maintainable. I also used google style sheet.